acceptance/acceptance_test.go

```diff
@@ -416,16 +418,33 @@ func formatOutput(w io.Writer, err error) {
 	}
 }
 
-func readIfExists(t *testing.T, path string) ([]byte, bool) {
+func tryReading(t *testing.T, path string) (string, bool) {
+	info, err := os.Stat(path)
+	if err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
+			t.Errorf("%s: %s", path, err)
```

Comment:
What's the benefit of using `t.Errorf` instead of returning the error from `tryReading` and logging it when the tryReading is called?

Reply:
In general the test helpers should not return errors but raise them directly on t object. This makes the usage less noisy w.r.t all the error checking.

In this particular case I'm using t.Error rather than t.Fatal because I want to see all the status/comparisons on all files.